src/ACEtk/details/verify/src/positive.hpp

```diff
@@ -0,0 +1,29 @@
+template< typename Range, 
+          utility::Require< true, utility::is_range, Range > = true >
+inline bool positive( const Range& array ){
```

Comment:
please begin predicate functions with is

Reply:
Yeah, forgot to change those.